### PR TITLE
Improve docker startup sequence

### PR DESCRIPTION
**Pull request by christian-heusel**
* **What kind of change does this PR introduce?** (Bug fix, feature, docs update, ...)
This bug is an enhancement and clears up some code in the entrypoint / docker files :blush:

* **Please check if the PR fulfills these requirements**
  - [ ] ~~E2E Tests for the changes have been added  via Cypress~~
  - [ ] ~~Meaningful rspec tests have been added~~
  - [ ] ~~Docs have been added / updated~~

* **What is the current behavior?** (You can also link to an open issue here)
  * The webpacker container runs the regular entrypoint ➡️ exceptions are thrown during boot!
  * every line in `entrypoint.sh` has the same pipe
  * environment variables for the tests are set

* **What is the new behavior (if this is a feature change)?**
  * The webpacker container does not try to execute the db migrations
  * the init functionality in `entrypoint.sh` is put in an extra file and the output piped once
  * the testcontainer has no external dependency (i.e. on `yarn install`)
  * 
* **Does this PR introduce a breaking change?** (What changes might users need to make in their application due to this PR?)
Nah 😄

### CODE REVIEW

**Comment by christian-heusel** [comment]
Why are the tests failing? 🤔

**Comment by christian-heusel** [comment]
https://github.com/MaMpf-HD/mampf/blob/bdf46e0e313d0026f88d201dc08403b80eae23ab/docker/run_cypress_tests/docker-compose.yml#L37-L39
Maybe this ToDo should be resolved to solve the failing tests

**Comment by christian-heusel** [comment]
Note that the CI failure is not a test failure 😃

**Review by henrixapp — APPROVED** [review]
Only thing i realized is, that you install yarn twice in the Dockerfile

**Comment by christian-heusel** [comment]
> Only thing i realized is, that you install yarn twice in the Dockerfile

In which dockerfile am i doing that? 👀

File: docker/development/Dockerfile [hunk 1]
```diff
@@ -12,21 +12,24 @@ RUN GOOS=js GOARCH=wasm go build -o pdfcomprezzor.wasm
...
 # https://github.com/nodesource/distributions#installation-instructions
 RUN curl -sL https://deb.nodesource.com/setup_10.x | bash - && \
     curl -sS https://dl.yarnpkg.com/debian/pubkey.gpg | apt-key add - && echo "deb https://dl.yarnpkg.com/debian/ stable main" | tee /etc/apt/sources.list.d/yarn.list && \
     curl -sS https://www.postgresql.org/media/keys/ACCC4CF8.asc | apt-key add - && echo "deb http://apt.postgresql.org/pub/repos/apt/ `lsb_release -cs`-pgdg main" | tee  /etc/apt/sources.list.d/pgdg.list && \
     sed -i "s/deb\.debian\.org/ftp.halifax\.rwth-aachen.de/g" /etc/apt/sources.list && \
-    apt-get update && apt-get install -y ffmpeg ghostscript imagemagick libarchive-tools nodejs pdftk postgresql-client-12 sqlite3 wget yarn --no-install-recommends && \
+    apt-get update && \
+    apt-get install --no-install-recommends -y ffmpeg ghostscript imagemagick \
+            libarchive-tools nodejs pdftk postgresql-client-12 sqlite3 wget \
+            wait-for-it yarn && \
```
Comment (henrixapp) [inline docker/development/Dockerfile:24]: Here!

File: docker/development/Dockerfile [hunk 2]
```diff
@@ -12,21 +12,24 @@ RUN GOOS=js GOARCH=wasm go build -o pdfcomprezzor.wasm
...
     rm -rf /var/lib/apt/lists/* && apt-get clean \
     && curl -o- -L https://yarnpkg.com/install.sh | bash
```
Comment (henrixapp) [inline docker/development/Dockerfile:26]: and here is the second install

File: docker/run_cypress_tests/Dockerfile [hunk 3]
```diff
@@ -12,14 +12,16 @@ RUN GOOS=js GOARCH=wasm go build -o pdfcomprezzor.wasm
...
 # https://github.com/nodesource/distributions#installation-instructions
 RUN curl -sL https://deb.nodesource.com/setup_10.x | bash - && \
     curl -sS https://dl.yarnpkg.com/debian/pubkey.gpg | apt-key add - && echo "deb https://dl.yarnpkg.com/debian/ stable main" | tee /etc/apt/sources.list.d/yarn.list && \
     curl -sS https://www.postgresql.org/media/keys/ACCC4CF8.asc | apt-key add - && echo "deb http://apt.postgresql.org/pub/repos/apt/ `lsb_release -cs`-pgdg main" | tee  /etc/apt/sources.list.d/pgdg.list && \
     sed -i "s/deb\.debian\.org/ftp.halifax\.rwth-aachen.de/g" /etc/apt/sources.list && \
-    apt-get update && apt-get install -y ffmpeg ghostscript imagemagick libarchive-tools nodejs pdftk postgresql-client-12 sqlite3 wget yarn --no-install-recommends && \
+    apt-get update && \
+    apt-get install -y --no-install-recommends \
+        ffmpeg ghostscript imagemagick libarchive-tools nodejs pdftk \
+        postgresql-client-12 sqlite3 wget wait-for-it yarn && \
```
Comment (henrixapp) [inline docker/run_cypress_tests/Dockerfile:24]: via apt

File: docker/run_cypress_tests/Dockerfile [hunk 4]
```diff
@@ -12,14 +12,16 @@ RUN GOOS=js GOARCH=wasm go build -o pdfcomprezzor.wasm
...
     rm -rf /var/lib/apt/lists/* && apt-get clean \
     && curl -o- -L https://yarnpkg.com/install.sh | bash
```
Comment (henrixapp) [inline docker/run_cypress_tests/Dockerfile:26]: from yarnpkg.com

**Pull request merged**